frontend/src/pages/book.scss

```diff
@@ -0,0 +1,85 @@
+@import '../fonts/fontWeight.scss';
+
```

Comment:
don't really like the floats and % unit
i would also like you to change the px unit into em, em is a unit scaled on the current font-size